custom_components/tahoma/sensor.py

```diff
@@ -87,7 +84,7 @@
     "core:TemperatureInCelsius": TEMP_CELSIUS,
     "core:TemperatureInKelvin": TEMP_KELVIN,
     "core:TemperatureInFahrenheit": TEMP_FAHRENHEIT,
-    "core:LuminanceInLux": UNIT_LX,
+    "core:LuminanceInLux": LIGHT_LUX,
```

Comment:
Don't forget to bump the minimum version of HA we need. Maybe we should just remove some other try catch import things soon and require the latest 2021.8 version.

Reply:
Oh I didn’t know it was recently added. In which version it was released ?

Reply:
Nevermind, it has been added on Sep 23, 2020... I thought this was also part of a recent commit where they refactored some of these units.

https://github.com/home-assistant/core/commit/6c8e0e20fbd0da03cc5a5d2561a94a4721367bf5